docs/GettingStarted.asciidoc

```diff
@@ -201,6 +201,49 @@ of the openQA security model, refer to the
 http://lizards.opensuse.org/2014/02/28/about-openqa-and-authentication/[detailed
 blog post] about the subject by the openQA development team.
 
+
+=== Job groups
+A job can belong to a job group. Those job groups are displayed on the index page
+and in the +Job Groups+ menu on the navigation bar. From there the job group overview
+pages can be accessed. Besides the test results the job group overview pages provide
+a description about the job group and allow commenting.
+
+Job groups have properties. These properties are mostly cleanup related. The
+configuration can be done in the operators menu for job groups.
+
+It is also possible to put job groups into categories. The nested groups will then
+inherit properties from the category. The categories are meant to combine job groups
+with common builds so test results for the same build can be shown together on
+the index page.
+
+
+=== Cleanup
+IMPORTANT: openQA automatically deletes data that it considers "old" based on
+different settings. For example job data is deleted from old jobs by the +gru+ task.
+
+The following cleanup settings can be done on job-group-level:
+
+[horizontal]
+size limit:: Limits size of assets
+keep logs for:: Specifies how long logs of a non-important job are retained after
+  it finished
+keep important logs for:: How long logs of an important job are retained after it
+  finished
+keep results for:: specifies How long results of a non-important job are retained
+  after it finished
+keep important results for:: How long results of an important job are retained after
+  it finished
+
+The defaults for those values are defined in
+https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Schema/JobGroupDefaults.pm[lib/OpenQA/Schema/JobGroupDefaults.pm].
+
+*NOTE* Deletion of job results includes deletion of logs and will cause the job to
+be completely removed from the database.
+
+Also note that jobs which do not belong to a job group are currently not affected by
```

Comment:
I would repeat the same format as in previous note:
`*NOTE*  jobs which do not ....`
instead of 'Also note'.